charts/ovn/templates/daemonset-controller-gw.yaml

```diff
@@ -66,8 +66,6 @@ spec:
           command:
             - /tmp/ovn-controller-init.sh
           env:
-            - name: OVN_CMS_OPTIONS
-              value: {{ .Values.conf.gw_ovn_cms_options | quote }}
```

Comment:
I think we need to restore this.

Reply:
this is already fixed. i think during your review, i made a new commit